src/devices/Camera/Camera/Settings/CommandOptionsBuilder.cs

```diff
@@ -255,6 +255,40 @@ public CommandOptionsBuilder WithMJPEGVideoOptions(int quality)
         return this;
     }
 
+    /// <summary>
+    /// Adds the option to capture a video stream at a certain Framerate.
+    /// </summary>
+    /// <param name="rate">The number of frames per second, if not specified uses the camera default</param>
+    /// <returns>A reference to this instance.</returns>
+    public CommandOptionsBuilder WithVideoFramerate(int rate)
```

Comment:
I simply copied one of the existing methods, it doesn't appear that UnitsNet is currently used at all in the Camera module. Would it be better to update them all in a separate PR instead of just introducing it for this one method?

Reply:
I didn't use UnitsNet because I wanted to maintain the numbers exactly as they are available in the command line of the apps.

_As soon as I have some time, I will review this PR as well. Sorry but I am very busy at the moment._